### ISSUE TITLE
error in calculating loss?

### ISSUE THREAD

**Issue by rrtucci** [dair-iitd/openie6#25]
model.py, line 182 reads:
loss = const_loss

Shouldn't this be 
loss += const_loss
